README.md

```diff
@@ -33,6 +34,24 @@ Install Node modules:
 pnpm install
 ```
 
+Start database server:
+
+```sh
+docker-compose up
+```
+
+Migrate the database:
+
+```sh
+pnpm migrate
```

Comment:
This didn't work for me. I believe this should be `pnpm migrate dev`. 

Running the above created another migration:

```
-- CreateIndex
CREATE INDEX "centroid_idx" ON "Area" USING GIST ("centroid");

-- CreateIndex
CREATE INDEX "limits_idx" ON "Area" USING GIST ("limits");
```

Did you forget to commit this or did I do something wrong?

Reply:
@oliverroick the schema was out of sync with the migrations and `pnpm migrate dev` tries to sync them when this happens. I've fixed it now.